mllib/src/test/scala/org/apache/spark/ml/feature/StopWordsRemoverSuite.scala

```diff
@@ -98,6 +98,7 @@ class StopWordsRemoverSuite
       .setInputCol("raw")
       .setOutputCol("filtered")
       .setStopWords(stopWords)
+      .setLocale("tr")
```

Comment:
I couldn't use special charset because of styles check, but I did it in Python's test.